app/controllers/notes_controller.rb

```diff
@@ -318,24 +313,16 @@ def liked
 
   def recent
     @title = I18n.t('notes_controller.recent_research_notes')
-    @wikis = Node.limit(10)
-      .where(type: 'page', status: 1)
-      .order('nid DESC')
-    @notes = Node.where(type: 'note', status: 1, created: Time.now.to_i - 1.weeks.to_i..Time.now.to_i)
-                 .order('created DESC')
-    @unpaginated = true
+    @pagy, @notes = pagy(published_notes.where(created: Time.now.to_i - 1.weeks.to_i..Time.now.to_i)
+                 .order('created DESC'))
     render template: 'notes/index'
   end
 
   # notes with high # of views
   def popular
     @title = I18n.t('notes_controller.popular_research_notes')
-    @wikis = Node.limit(10)
-      .where(type: 'page', status: 1)
-      .order('nid DESC')
-    @notes = Node.research_notes
+    @notes = published_notes
       .limit(20)
```

Comment:
seems like here the `.order` should come before the `limit`

Reply:
Yes, good catch, I'll add that

Reply:
I've just realized that the order method is from the [@wiki ](https://github.com/publiclab/plots2/pull/8800/files/c5954b67e9b5c1e72d6e6192a73883f167362c87#diff-c254740097a8c0601832077488db77e572b2c2b926f77508fadc6081b0052804L333)query, but it's true that the order should come before the limit in order to get the most recent ones which is also what @Sagarpreet has suggested.

Reply:
I've added the order in the [published_notes](https://github.com/publiclab/plots2/pull/8800/files#diff-c254740097a8c0601832077488db77e572b2c2b926f77508fadc6081b0052804R454) method